README.md

```diff
@@ -0,0 +1,12 @@
+Mender Test Containers
+==============================================
+
+This repository contains docker image definitions needed to continously build and test products for the Mender company.
+
+The images in here are used in the following situations:
+
+| docker tag | folder | usage example |
+|---|---|---|
+| raspbian_latest | docker | ??? |
```

Comment:
It is used in this very same repo. See https://github.com/mendersoftware/mender-test-containers/blob/2b43f2290ba1395eb4f094b2b2c582c749899259/container_props.py#L51

Reply:
🙈 argh thanks! I guess I just interpreted the search result from the same repo as part of the build...